src/devices/purmo.ts

```diff
@@ -0,0 +1,66 @@
+import fz from '../converters/fromZigbee';
```

Comment:
Please also add to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/index.ts